### PR TITLE
[DOCS] Fix formatting issue in cardinality-aggregation.asciidoc

### PR DESCRIPTION
**Pull request by abdonpijpelink**
Fixes several issues with the "Execution hint" section.

### CODE REVIEW

File: docs/reference/aggregations/metrics/cardinality-aggregation.asciidoc [hunk 1]
```diff
@@ -241,6 +241,7 @@ POST /sales/_search?size=0
 --------------------------------------------------
 // TEST[setup:sales]
 <1> Documents without a value in the `tag` field will fall into the same bucket as documents that have the value `N/A`.
+
 ==== Execution Hint
```
Comment (abdonpijpelink) [inline docs/reference/aggregations/metrics/cardinality-aggregation.asciidoc:245]:
```suggestion
==== Execution hint
```

File: docs/reference/aggregations/metrics/cardinality-aggregation.asciidoc [hunk 2]
```diff
@@ -241,9 +241,10 @@ POST /sales/_search?size=0
 --------------------------------------------------
 // TEST[setup:sales]
 <1> Documents without a value in the `tag` field will fall into the same bucket as documents that have the value `N/A`.
-==== Execution Hint
 
-There are different mechanisms by which cardinality aggregations can be executed:
+==== Execution hint
+
+There are different mechanisms by which cardinality aggregations can be run:
```
Comment (jrodewig) [inline docs/reference/aggregations/metrics/cardinality-aggregation.asciidoc:247]:
```suggestion
You can run cardinality aggregations using different mechanisms:
```

**Review by jrodewig — APPROVED** [review]
Minor nit you can ignore if wanted. LGTM. Thanks!

**Comment by abdonpijpelink** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.9|[](https://github.com/elastic/elasticsearch/pull/98252)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**